packages/ui-components/src/components/info-label/info-label.base.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../assets/styles/globals';
+
+/**
+	 * @prop --text-color-title: Info label's title color.
+	 * @prop --text-color-description: Info label's description color.
+*/
+
+:host {
+	--text-color-title: #{kv-color('neutral-4')};
+	--text-color-description: #{kv-color('neutral-0')};
+}
+
+.info-label {
+	display: flex;
+	flex-direction: column;
+
+	.title {
+		@include kv-font-label-small-uppercase-bold();
+		color: var(--text-color-title);
+		white-space: nowrap;
+	}
+
+	.description-wrapper {
+		margin-top: $spacing;
+		overflow-y: hidden;
+		transition: height 0.3s ease-in-out;
+		height: auto;
+
+		.text {
+			@include kv-font-span-regular();
+			color: var(--text-color-description);
+		}
+	}
+
+	.expand-description-button {
+		@include kv-font-h5-semibold();
+		display: flex;
+		align-items: center;
+		cursor: pointer;
+		margin-top: $spacing-4x;
+		color: kv-color('primary');
```

Comment:
Probably is better to put the expanded button color in variable, like we does for title and description